templates/control

```diff
@@ -29,23 +29,36 @@ Depends: apache2 | apache | apache-ssl | apache-perl | httpd,
     logrotate,
     mysql-client,
     mysql-server,
+    libjs-jquery-timepicker,
+    libjs-bootstrap,
+    libjs-jquery-cookie,
+    libjs-jquery-ui,
+    libjs-jquery-ui-theme-ui-lightness,
+    tinymce,
```

Comment:
For this version version of glpi, tinymce is strongly patched (custom drag & drop support).
I did the clean work in the upcoming version (9.1).

It's a good enhancement but a little too early :)